cf-testd/cf-testd.c

```diff
@@ -24,62 +25,60 @@
 
 #include <platform.h>
 
-#include <client_code.h>                // cfnet_init
-#include <crypto.h>                     // CryptoInitialize
+#include <client_code.h> // cfnet_init
+#include <crypto.h>      // CryptoInitialize
 
+#include <bootstrap.h> // GetAmPolicyHub
 #include <cf-serverd-enterprise-stubs.h>
+#include <conversion.h> // MapAddress
+#include <exec_tools.h> // ActAsDaemon
+#include <item_lib.h>   // DeleteItemList
+#include <known_dirs.h> // GetInputDir
+#include <loading.h>    // LoadPolicy
+#include <locks.h>      // AcquireLock
+#include <man.h>        // ManPageWrite
+#include <mutex.h>      // ThreadLock
 #include <net.h>
+#include <openssl/err.h>   // ERR_get_error
+#include <policy_server.h> // PolicyServerReadFile
+#include <printsize.h>     // PRINTSIZE
+#include <server_access.h> // acl_Free
+#include <server_code.h>   // InitServer
 #include <server_common.h>
-#include <server_access.h>              // acl_Free
-#include <item_lib.h>                   // DeleteItemList
-#include <server_transform.h>           // Summarize
-#include <bootstrap.h>                  // GetAmPolicyHub
-#include <policy_server.h>              // PolicyServerReadFile
-#include <systype.h>                    // CLASSTEXT
-#include <mutex.h>                      // ThreadLock
-#include <locks.h>                      // AcquireLock
-#include <exec_tools.h>                 // ActAsDaemon
-#include <man.h>                        // ManPageWrite
-#include <server_tls.h>                 // ServerTLSInitialize
-#include <tls_generic.h>                // TLSLogError
-#include <timeout.h>                    // SetReferenceTime
-#include <known_dirs.h>                 // GetInputDir
-#include <sysinfo.h>                    // DetectEnvironment
-#include <time_classes.h>               // UpdateTimeClasses
-#include <loading.h>                    // LoadPolicy
-#include <printsize.h>                  // PRINTSIZE
-#include <conversion.h>                 // MapAddress
-#include <server_code.h>                // InitServer
-#include <signals.h>                    // ReloadConfigRequested
-#include <openssl/err.h>                // ERR_get_error
+#include <server_tls.h>       // ServerTLSInitialize
+#include <server_transform.h> // Summarize
+#include <signals.h>          // ReloadConfigRequested
+#include <sysinfo.h>          // DetectEnvironment
+#include <systype.h>          // CLASSTEXT
+#include <time_classes.h>     // UpdateTimeClasses
+#include <timeout.h>          // SetReferenceTime
+#include <tls_generic.h>      // TLSLogError
```

Comment:
I liked this better before.

Reply:
Do you mean the spaces before comments or the ordering?

It seems googles default style orders includes alphabetically. I can turn that off. If you like more spaces before the comments that can be adjusted, but I like just 1.

Reply:
The order doesn't really matter to me. But I like when the comments are all aligned to the column not colliding with any non-comment characters on other lines. i.e, just like it was before.